packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/main/index.md

```diff
@@ -1,48 +1,52 @@
+In Identity Engine, you can use authenticator-based authenticator enrollment policies to govern recovery authenticator enrollment for the password recovery flow. This feature isn't available for factor-based authenticator enrollment policies.
+
+The enrollment for recovery authenticators (Email, Phone, Okta Verify, and Security question) are governed by both the password policy and the authenticator enrollment policy. For example, if the Email authenticator is enabled and set to `Required` in the authenticator enrollment policy, then email enrollment is required for recovery even if it isn't required in the password policy.
 
-> **Note:** Password policy recovery authenticator settings supersedes the authenticator settings in an MFA enrollment policy. For example, if the Phone authenticator is `Optional` or `Disabled` for the MFA enrollment policy, but `Required` for the password policy, then phone enrollment is required for the password recovery flow.
+> **Note:** Password policy recovery authenticator settings supersedes authenticator enrollment policy settings. For example, if the Phone authenticator is `Optional` or `Disabled` for the authenticator enrollment policy, but `Required` for the password policy, then phone enrollment is required for the password recovery flow.
 
 See [password policy](/docs/reference/api/policy/#password-policy) and [Configure password policies](https://help.okta.com/okta_help.htm?type=oie&id=csh-configure-password) for details on how to configure the password policy in the Admin Console.
 
-## Get MFA enrollment policies
+## Get authenticator enrollment policies
 
-To parse a response from the `GET /api/v1/policies?type=MFA_ENROLL` request, you must determine if the returned MFA enrollment policy contains either `authenticators` or `factors` in the [settings schema](/docs/reference/api/policy/#policy-settings-data-2).
+To parse a response from the `GET /api/v1/policies?type=MFA_ENROLL` request, you must determine if the returned authenticator enrollment policy contains either `authenticators` or `factors` in the [settings schema](/docs/reference/api/policy/#policy-settings-data-2).
 
-> **Note:** Another indicator for an authenticator MFA enrollment policy is that the [settings schema](/docs/reference/api/policy/#policy-settings-data-2) contains a `type` property that is set to the `AUTHENTICATORS` value.
+> **Note:** The `type` property in the [settings schema](/docs/reference/api/policy/#policy-settings-data-2) is another indicator of an authenticator-based or factor-based enrollment policy.
```

Comment:
Again, not sure if necessary or if this should be put in another place but the type attribute is not always available. It is always available when it is authenticators and it can also be safely assumed the factor settings are being used if the type attribute is not available.

Reply:
Thanks for the clarification. In the API reference we have `type=FACTORS` as default. This seems to suggest that `type` would always be present in an authenticator enrollment policy. Can you please confirm? Maybe we need to remove that default description?
If `type` is not always available, then I'll update the note to: 
> **Note:** Another indicator of an authenticator-based enrollment policy is when`type=AUTHENTICATORS` in the [settings schema](/docs/reference/api/policy/#policy-settings-data-2).

Reply:
Type is not always available, like if an org gets migrated to OIE without any changes than type will not be available. Also in the API it is possible to just create a policy without the type if they're using factors settings.